CHANGES.md

```diff
@@ -1,3 +1,28 @@
+v4.0.0
+------
+
+- Sexplib is now an optional library for the base `Cstruct` module.
+  A new `Cstruct_sexp` module has been introduced with the serialiser
+  functions, contained within the `cstruct-sexp` opam package.
```

Comment:
The ppx will emit code that depends on sexplib when using `[@@sexp]`. I think that this should be documented somewhere. A potential source of confution is that it sets `(ppx_runtime_libraries cstruct)` only - this is probably what we want, but it will miss the `sexplib` dependency if for some reason the generated code is the only use of sexplib in a module.

Reply:
Yes I noticed this in the porting of the tests. The workaround was pretty easy (just depend on sexplib), but I'll note this explicitly in the README (which I have to update, just remembered I only did CHANGES so far)